.github/workflows/test.yml

```diff
@@ -30,14 +30,14 @@ jobs:
         with:
           node-version: 20
           check-latest: false
-      - name: node_modules cache
-        uses: actions/cache@v3
-        id: node_modules_cache
-        with:
-          path: ./node_modules
-          key: ${{ runner.os }}-20-${{ hashFiles('package-lock.json') }}
-          restore-keys: |
-            ${{ runner.os }}-20-
+      # - name: node_modules cache
```

Comment:
Is this intentional?

Reply:
Yeah it's a bit of a mess right now sorry - I'm running into some branch protection rules. is the `adapter-nextjs-*` branch protection rule still relevant, or can it be deleted?

Reply:
Feel free to delete